integration/test_auth.py

```diff
@@ -228,8 +228,14 @@ def test_list_users(new_user_list):
             assert isinstance(user, auth.ExportedUserRecord)
             if user.uid in new_user_list:
                 fetched.append(user.uid)
-                assert user.password_hash is not None
-                assert user.password_salt is not None
+                assert user.password_hash is not None, (
+                    "Missing password_hash field. A common cause would be forgetting to add the " +
```

Comment:
Nit: Extract the repeated part into  constant.

Reply:
Done.